frontend/src/app/app.component.ts

```diff
@@ -3,6 +3,7 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
```

Comment:
This is the line that can be removed once the `app.component.scss` files is deleted.

Reply:
I have removed this line.